pkg/virt-handler/vm.go

```diff
@@ -2425,8 +2425,8 @@ func (d *VirtualMachineController) helperVmShutdown(vmi *v1.VirtualMachineInstan
 		return err
 	}
 
-	// Only attempt to gracefully shutdown if the domain has the ACPI feature enabled
-	if isACPIEnabled(vmi, domain) && tryGracefully {
+	// Only attempt to gracefully shutdown if tryGracefully is true
+	if tryGracefully {
```

Comment:
Don't we still have to verify that ACPI is enabled for non-s390x VMs?

Reply:
@nekkunti Why did you mark this as resolved?

Reply:
sorry .. my bad , my reply was in Pending state ( forgot to submit reply) .

Reply:
I don't see it there?
Can you link the exact line, e.g. https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/manager.go#L1802?